playbooks/deploy.yaml

```
@@ -51,9 +49,9 @@
       - name: candlepin
         password: "{{ candlepin_db_password }}"
```

If you're changing things, can you make this consistent
```suggestion
        password: "{{ candlepin_database_password }}"
```

---

Yea - I want to settle https://github.com/theforeman/foreman-quadlet/pull/134#discussion_r2056242787 before I update them as that will likely dictate the naming scheme.